File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        list1 = [1]
        if rowIndex == 0:
            return list1
        row = 0
        while row<rowIndex:
            list2 = []
            list2.append(1)
            for i in range(len(list1)-1):
                list2.append(list1[i]+list1[i+1])
            list2.append(1)
            list1 = list2
            row+=1
        return list2

File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> list1 = new ArrayList<>();
        list1.add(1);
        while(rowIndex>0){
            List<Integer> list2= new ArrayList<>();
            list2.add(1);
            for(int i=0;i<list1.size()-1;i++)
                list2.add(list1.get(i)+list1.get(i+1));
            list2.add(1);
            list1 = list2;
            rowIndex--;
        }
        return list1;
    }
}